ghost/core/test/e2e-webhooks/pages.test.js

```diff
@@ -82,6 +82,57 @@ describe('page.* events', function () {
         mockManager.restore();
     });
 
+    it('page.published event is triggered', async function () {
+        const webhookURL = 'https://test-webhook-receiver.com/page-published/';
+        await webhookMockReceiver.mock(webhookURL);
+        await fixtureManager.insertWebhook({
+            event: 'page.published',
+            url: webhookURL
+        });
+
+        const res = await adminAPIAgent
+            .post('pages/')
+            .body({
+                pages: [
+                    {
+                        title: 'testing page.published webhook',
```

Comment:
The snapshot tests here are failing most likely because the post title is a bit long and the "anyLocalUrl" matcher did not handle it properly (here is a fix for it [form another PR](https://github.com/TryGhost/Ghost/pull/15642/files/58e615301406db54d0cff1301b5c29cd70ae8bcf#diff-bba0a1186052539ef90dbfef638bf93c62a3744e46fa0b1eb28b7f2d4758a4c7R41)). to make things easier, might just give the page a single-word title OR reuse the fix from the linked above PR